modin/pandas/test/test_io.py

```diff
@@ -375,15 +445,13 @@ def test_read_csv_parsing_2(
             fn_name="read_csv",
             check_exception_type=None,  # issue #2320
             raising_exceptions=None,
-            check_kwargs_callable=not callable(skiprows),
+            cast_to_str=True,
```

Comment:
Why cast to string is needed?

Reply:
This line was removed in the last commit.